src/containers/EventsList.js

```diff
@@ -25,7 +25,29 @@ export class EventsList extends Component {
     return (
       <Fragment>
         <Container>
-          <Header as='h1'>AgileVentures Events</Header>
+          <Grid>
+            <Grid.Row>
+              <Grid.Column width={14}>
+                <Header as='h1'>
+                 AgileVentures Events
+                </Header>
+              </Grid.Column>
+              <Grid.Column width={1}>
+                <a href='events/new'>
+                  <Popup
+                    position='right center'
+                    trigger={
+                      <Button basic style={{ marginTop: '25px' }}>
+                        <Icon name='plus' id='new-proj-icon' />
```

Comment:
Can you please change the id to be unique to the `new-event-icon`?

Reply:
I updated the `id`